cmd/generate/config/rules/privatekey.go

```diff
@@ -11,18 +9,33 @@ func PrivateKey() *config.Rule {
 	r := config.Rule{
 		Description: "Private Key",
 		RuleID:      "private-key",
-		Regex:       regexp.MustCompile(`(?i)-----BEGIN[ A-Z0-9_-]{0,100}PRIVATE KEY( BLOCK)?-----[\s\S-]*KEY----`),
+		Regex:       generateUniqueTokenRegex(`(?i)-----BEGIN[ A-Z0-9_-]{0,100}PRIVATE KEY-----\n[0-9a-zA-Z+\/]{64,512}={0,2}[\s\S-]*KEY----(?i)`),
```

Comment:
My thought was, it would be good to capture the entire private key rather than just the beginning header maybe? Which was what you suggested `generateUniqueTokenRegex` could do over `regexp.MustCompile`? 
```
Secret:      -----BEGIN OPENSSH PRIVATE KEY-----  
```